test/setup.js

```diff
@@ -0,0 +1,83 @@
+
+/**
+ * Dependencies
+ */
+
+const babelConfig = require('../package.json').babel;
+const register = require('babel-core/register');
+const chaiEnzyme = require('chai-enzyme');
+const path = require('path');
+const chai = require('chai');
+const fs = require('fs');
+
+// only babel transpile these node_modules:
+const modulesToCompile = [
+  'react-native',
+  'react-native-mock'
+].map((moduleName) => new RegExp(`/node_modules/${moduleName}`));
+
+babelConfig.ignore = function(filename) {
+  if (!(/\/node_modules\//).test(filename)) return false;
+  const matches = modulesToCompile.filter((regex) => regex.test(filename));
+  const shouldIgnore = matches.length === 0;
+  return shouldIgnore;
+}
+
+register(babelConfig);
+
+// setup globals / chai
+global.__DEV__ = true;
+global.expect = chai.expect;
+chai.use(chaiEnzyme());
+
+// setup mocks
+require('react-native-mock/mock');
+const React = require('react-native')
+
+// add modules missing from react-native-mock
+React.Alert = { alert: React.View };
+React.UIManager.setLayoutAnimationEnabledExperimental = () => {};
+React.NavigationExperimental = {
+  AnimatedView: React.View
+};
+
+/**
+ * HACK: support .android.js and .ios.js
+ * in Node test runner environment.
+ */
+
+var Module = require('module');
+var originalRequire = Module.prototype.require;
+var platform = process.env.PLATFORM || 'ios';
```

Comment:
Easiest way would be to run the test suite twice, once for `ios` and once for `android`.